src/main/fc/config.h

```diff
@@ -36,7 +36,7 @@ typedef enum {
     FEATURE_VBAT = 1 << 1,
     FEATURE_TX_PROF_SEL = 1 << 2,       // Profile selection by TX stick command
     FEATURE_BAT_PROFILE_AUTOSWITCH = 1 << 3,
-    FEATURE_UNUSED_12 = 1 << 4,  //was FEATURE_MOTOR_STOP
+    FEATURE_GEOZONE = 1 << 4,  //was FEATURE_MOTOR_STOP
```

Comment:
Any reason to re-use this, instead of adding a new one? Have we run out of feature bits?

Reply:
Yes, all 32 bits are used.